src/Gui/DlgAbout.cpp

```diff
@@ -216,6 +216,11 @@ QPixmap AboutDialog::aboutImage() const
     }
 
     std::string about_path = App::Application::Config()["AboutImage"];
+    // See if current version is dev
+    QString suffix  = QString::fromLatin1(App::Application::Config()["BuildVersionSuffix"].c_str());
+    if (suffix == QString::fromLatin1("dev")) {
+        about_path = App::Application::Config()["AboutImageDev"];
+    }
```

Comment:
No need to use `QString` here, as everything is`std::string` already.
This can greatly simplify the code to something like:

```c++
const auto suffix = App::Application::Config()["BuildVersionSuffix"];
const auto about_image_key = (suffix == "dev") ? "AboutImageDev"
                                               : "AboutImage";
const auto about_path = App::Application::Config()[about_image_key];
```